README.md

```diff
@@ -112,10 +175,19 @@ Also, githooks can be installed to format the code per-commit:
 
 ## Using docker
 
-The easiest way to setup the development environment is to use docker. You can build the top-level docker file:
+The easiest way to setup the development environment is to use docker. Docker files for different versions of ROCm are
+provided. ```hcc.docker``` is for ROCm3.3, and ```hip-clang.docker``` is for ROCm3.7.
```

Comment:
Users should just use the top-level Dockerfile. hcc is deprecated and these `hcc.docker` and `hip-clang.docker` will be removed in the future.

Reply:
Containers built from these two files work fine, while a container built from the Dockfile does not work, some components are missed.

Reply:
Why does the Dockerfile not work? We use the Dockerfile on our CI, so it does work.